cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/azure-mc-common/management-credential-step/ManagementCredentials.tsx

```diff
@@ -120,6 +121,52 @@ function ManagementCredentials(props: Partial<StepProps>) {
             }
         }
     };
+
+    const isConnected = () => {
+        if (connectionStatus === CONNECTION_STATUS.CONNECTED) {
+            return true;
+        }
+        return false;
+    };
+
+    const hasError = () => {
+        if (Object.keys(errorObject).length > 0) {
+            return true;
+        }
+        return false;
+    };
+
+    function showErrorInfo() {
+        if (isConnected() && hasError()) {
+            return (
+                <div>
+                    <div className="error-text">Error Occurred</div>
+                    <br />
+                    {Object.keys(errorObject).map((errorField) => {
+                        return (
+                            <CdsControlMessage status="error" key={errorField}>
+                                {errorObject[errorField].status}
+                                &nbsp;
+                                {errorObject[errorField].statusText}
+                                &nbsp;
+                                {errorObject[errorField].body.message}
+                                <br />
+                            </CdsControlMessage>
+                        );
+                    })}
+                    <br />
+                </div>
+            );
+        }
+        return;
+    }
+
+    useEffect(() => {
+        if (azureState[STORE_SECTION_FORM][AZURE_FIELDS.REGION]) {
```

Comment:
I'm wondering if it's possible for a user to `unselect` the REGION. If so, we probably want to clear the resources associated with it (including the os images). So it may be good to all `initOsImages` when there is no REGION and have it clear the os images.

Reply:
Yes sir. Will implement that.